Frontend/android-native/app/src/user/jatriSaathi/assets/juspay/v1-configuration.js

```diff
@@ -291,6 +291,7 @@ window.getMerchantConfig = function () {
       }
   , "showDisabilityBanner" : false
   , "showCheckoutRentalBanner" : true
+  , "enableHelpAndSupport" : false
```

Comment:
Is it different support. There is already a key present to enable and disable support. can we reuse that?